src/Event/FastEvmProxy.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Penny\Event;
+
+use FastEventManager\EventManager;
+
+class FastEvmProxy implements EventManagerInterface
```

Comment:
This is just a proxy in Penny domain, but it is called FastEvmProxy. IMHO Proxy or EventProxy should be enough.